_includes/pbc/all/install-features/202400.0/marketplace/install-the-marketplace-merchant-portal-product-offer-service-points-feature.md

```diff
@@ -0,0 +1,179 @@
+
+
+This document describes how to integrate the Marketplace Merchant Portal Product Offer Service Points feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Marketplace Merchant Portal Product Offer Service Points feature core.
+
+### Prerequisites
+
+Install the required features:
+
+| NAME                         | VERSION          | INTEGRATION GUIDE                                                                                                                                                                                          |
+|------------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace Merchant Portal Core | {{page.version}}  | [Merchant Portal Core feature integration](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-the-marketplace-merchant-portal-core-feature.html)   |
+| Marketplace Merchant Portal Product Offer Management    | {{page.version}} | [Marketplace Product Offer feature integration](/docs/pbc/all/offer-management/{{page.version}}/marketplace/install-and-upgrade/install-the-marketplace-merchant-portal-product-offer-management-feature.html)                        |
+| Marketplace Product Offer + Service Points | {{page.version}} | [Install the Marketplace Product Offer + Service Points feature](/docs/pbc/all/offer-management/{{page.version}}/marketplace/install-and-upgrade/install-the-marketplace-product-offer-service-points-feature.html)             |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/product-offer-service-point-merchant-portal-gui:"{{page.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| ProductOfferServicePointMerchantPortalGui | spryker/product-offer-service-point-merchant-portal-gui |
```

Comment:
Let's add the following modules, because we are mentioning the plugins from them:
* ProductOfferServicePoint
* ServicePoint

Reply:
We have the corresponding features in prerequisites where we check the modules existence, before installing this feature a user must go throw the corresponding IGs, it doesn't makes sense to repeat in this IG (the same for transfers, etc)